### PR TITLE
[test-infra] Update to podutils (drop bootstrap.py)

### PR DESCRIPTION
**Pull request by dims**
related to https://github.com/kubernetes/test-infra/issues/30846

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/janitors.yaml [hunk 1]
```diff
@@ -82,15 +82,15 @@ periodics:
...
     preset-service-account: "true"
+  decorate: true
+  decoration_config:
+    timeout: 30m
   spec:
     containers:
-    - args:
-      - --bare
-      - --timeout=30
-      - --service-account=/etc/service-account/service-account.json
```
Comment (aojea) [inline config/jobs/kubernetes/test-infra/janitors.yaml:90]: this flag for the service account  👀 , @BenTheElder @ameukam

Comment (dims) [reply config/jobs/kubernetes/test-infra/janitors.yaml:90]: automatically injected

Comment (aojea) [reply config/jobs/kubernetes/test-infra/janitors.yaml:90]: the janitor job scares me if something goes wrong, will defer to others more familiars

Comment (dims) [reply config/jobs/kubernetes/test-infra/janitors.yaml:90]: we're gonna try it out and file a revert if needed

**Pull request merged**